internal/sql/repository/CiArtifactRepository.go

```diff
@@ -189,7 +189,7 @@ func (impl CiArtifactRepositoryImpl) GetArtifactsByCDPipeline(cdPipelineId int)
 			}
 		}
 		artifactsAll = append(artifactsAll, a)
-		if len(artifactsAll) >= 10 {
+		if len(artifactsAll) >= 30 {
```

Comment:
should we take this number from outside

Reply:
in request parameter? we can, but that will add work on frontend side. Maybe we can update this in future release